platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r812/ProblemThresholdCrossVersionTest.groovy

```diff
@@ -115,7 +115,7 @@ class ProblemThresholdCrossVersionTest extends ToolingApiSpecification {
         given:
         def exceedingCount = 2
         def differentProblemCount = 4
-        def threshold = THRESHOLD_DEFAULT_VALUE + exceedingCount
+        def threshold = threshold + exceedingCount
```

Comment:
Shouldn't it be `ProblemThresholdCrossVersionTest.threshold`? But if it works, okay.

Reply:
It works correctly, but the readability is indeed poor. I'll try to make the static field upper-case. IIRC I had some troubles with `sanityCheck`; let's see.

Reply:
Violation: Rule=FieldName P=2 Loc=.(ProblemThresholdCrossVersionTest.groovy:38) Msg=[The fieldname THRESHOLD in class org.gradle.integtests.tooling.r812.ProblemThresholdCrossVersionTest does not match [a-z][a-zA-Z0-9]*] Src=[private static def THRESHOLD = 15]

Reply:
May be it's the rule that should be fixed 🤔

Reply:
@ov7a ptal https://github.com/gradle/gradle/pull/32040